AirCastingTests/SDSyncFileWritingServiceTests.swift

```diff
@@ -10,31 +10,59 @@ class SDSyncFileWritingServiceTests: ACTestCase {
         let threshold = 3
         let service = SDSyncFileWritingService(bufferThreshold: threshold)
         let testWritesCount = 110
+        let uuid = "123"
+        let uuid2 = "456"
```

Comment:
those tests need a major cleanup. They work and test stuff, but are very hard to read. For tests it's lethal since the purpose of unit testing is fast feedback - if we can't tell what's failing just by looking at test name it doesn't really do its job

Reply:
I would like to send this code to Michael for testing and then clean up this test and write some more tests while he is doing his testing. Is that okay for you?